Documentation/kubernetes-on-aws-render.md

```diff
@@ -0,0 +1,255 @@
+# Configure your Kubernetes cluster on AWS
+
+This is the [second step of running Kubernetes on AWS][aws-step-1]. Before we launch our cluster, let's define a few parameters that the cluster requires.
+
+## Cluster parameters
+
+### EC2 key pair
+
+The keypair that will authenticate SSH access to your EC2 instances. A single key will be configured on each CoreOS node for debugging any issues you may encounter.
+
+Keep in mind that key pairs are tied to a region. More info in the [EC2 Keypair docs](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html).
```

Comment:
Add: After creating a key pair, you will use the name you gave the keys to configure the cluster.